test/data/validation/destinations/intercom_v2.json

```diff
@@ -0,0 +1,105 @@
+[
```

Comment:
try adding some tests around the destination specific fields like `apiServer` too